### PR TITLE
Enable Layout/AlignHash rubocop

### PR DESCRIPTION
**Pull request by coolo**
In a codebase that is otherwise auto aligned, I find the super inconsistent hash indentation pretty annoying. Sometimes it's 2 chars left, sometimes 3 right sometimes even inconsistent in itself.

So @openSUSE/open-build-service - this cop does autocorrect, so it's a simple vote, no extra work required.

### CODE REVIEW

**Comment by coolo** [comment]
ok, haml_lint is also using this cop, but does not support auto correct. So there are exceptions :)

**Comment by coolo** [comment]
Filed an issue upstream, so we can take out the config on update

File: src/api/.rubocop.yml [hunk 1]
```diff
@@ -224,3 +224,7 @@ Style/MethodCallWithArgsParentheses:
     - write
     - yield
     - ymp
+
+Layout/AlignHash:
+  Enabled: true
+
```
Comment (eduardoj) [inline src/api/.rubocop.yml]: Nitpick: as the cop is enabled by default this lines are not needed.

**Comment by eduardoj** [comment]
Fine for me, too. :+1:

File: src/api/.rubocop.yml (hunk 1 above)
Comment (Ana06) [reply src/api/.rubocop.yml]: As @eduardoj says this is enabled by default. But removing it is IMO not a nitpick. We have already agreed to follow the defaults for this cop, which is not what is done if you add it here.

File: src/api/app/controllers/application_controller.rb [hunk 2]
```diff
@@ -192,7 +192,7 @@ def pass_to_backend(path = nil)
 
     text = response.body
     send_data(text, type: response.fetch('content-type'),
-      disposition: 'inline')
+                    disposition: 'inline')
```
Comment (Ana06) [inline src/api/app/controllers/application_controller.rb]: Is this correct? :thinking:

File: src/api/app/helpers/webui/buildresult_helper.rb [hunk 3]
```diff
@@ -61,7 +61,7 @@ def webui2_arch_repo_table_cell(repo, arch, package_name, status = nil, enable_h
     capture_haml do
       if enable_help && status['code']
         concat(content_tag(:i, nil, class: ['fa', 'fa-question-circle', 'text-info', 'mr-1'],
-                           data: { content: Buildresult.status_description(status['code']), placement: 'top', toggle: 'popover' }))
+                                    data: { content: Buildresult.status_description(status['code']), placement: 'top', toggle: 'popover' }))
```
Comment (Ana06) [inline src/api/app/helpers/webui/buildresult_helper.rb]: same here... this looks weird...

**Comment by Ana06** [comment]
@coolo 

> So @openSUSE/open-build-service - this cop does autocorrect, so it's a simple vote, no extra work required.

No vote required, as it is said in [our Wiki](https://github.com/openSUSE/open-build-service/wiki/Code-Style):

> the offenses in .rubocop_todo.yml, are rules we have already agreed on but that we didn't solve yet (most likely because we didn't have time). That means that those offenses can be solved without a previous discussion.

**Comment by coolo** [comment]
aehm, @Ana06 - I read 'Enabled: false' not as an offense but as a diabled cop

**Comment by coolo** [comment]
I took out the extra enable - but I still think this requires a vote (even though I have a hard time imaging veto arguments)

**Comment by bgeuken** [comment]
Fine with me:thumbsup:

**Pull request merged**